.vscode/launch.json

```diff
@@ -18,82 +21,108 @@
+            "request": "launch",
+            "program": "${workspaceFolder}/build-BellHybrid-linux-Debug/BellHybrid.elf",
+            "args": [],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}/build-BellHybrid-linux-Debug/",
+            "environment": [],
+            "externalConsole": false,
             "MIMode": "gdb",
-            "processId": "${command:pickProcess}",
             "setupCommands": [
                 {
                     "description": "Enable pretty-printing for gdb",
                     "text": "-enable-pretty-printing",
                     "ignoreFailures": true
                 },
-                {
-                    "description": "disable SIGUSR1 logs",
-                    "text": "handle SIGUSR1 nostop noprint"
-                },
+                // {
+                //     "description": "disable SIGUSR1 logs",
+                //     "text": "handle SIGUSR1 nostop noprint"
+                // }
             ]
         },
+
+        // RT1051
+
         {
-            "name": "Debug (rt1051)",
-            "type": "gdb",
-            "gdbpath": "arm-none-eabi-gdb",
-            "request": "attach",
-            "target": ":2331",
-            "remote": true,
-            "cwd": "${workspaceRoot}",
-            "valuesFormatting": "parseText",
-            "executable": "${workspaceFolder}/build-rt1051-Debug/PurePhone.elf",
-            "autorun": [
-                "eval \"monitor exec SetRTTAddr %p\", &_SEGGER_RTT",
+            "name": "Launch PurePhone (RT1051)",
+            "type": "cortex-debug",
+            "request": "launch",
+            "executable": "${workspaceFolder}/build-PurePhone-rt1051-RelWithDebInfo/PurePhone.elf",
+            "servertype": "jlink",
+            "serverpath": "JLinkGDBServerCLExe",
+            "cwd": "${workspaceFolder}",
+            "gdbPath": "arm-none-eabi-gdb",
+            // "debuggerArgs": ["-x", "${workspaceFolder}/.gdbinit-1051"],
+            // "serverArgs": ["-strict", "-ir", "-singlerun", "-speed", "25000"],
+            // "runToEntryPoint": "main",
+            // "runToMain": true,
+            "preLaunchCommands": [],
+            "postLaunchCommands": [],
+            "interface": "swd",
+            "device": "MCIMXRT1051",
+            "jlinkscript": "${workspaceFolder}/evkbimxrt1050_sdram_init.jlinkscript",
+            "rtos": "FreeRTOS",
+            "overrideLaunchCommands": [
+                // "source .gdb_macros",
+                // "handle SIGUSR1 nostop noprint",
+                // "maint set internal-warning quit no",
+                // "maint set internal-warning corefile no",
+                // "set pagination off",
+                // "set python print-stack full",
+                // "target remote localhost:2331",
+                // "source tools/gdb_crash_extend.py",
+                // "source tools/misc/puregdb/puregdb.py",
```

Comment:
No, but these lines are taken from `run.sh` so in case we needed them in the future I left them in the config file. In case I didn't account for something.